Jenkinsfile

```diff
@@ -479,6 +479,10 @@ pipeline {
             name: "TARGET_GFX94X",
             defaultValue: false,
             description: "")
+        booleanParam(
+            name: "TARGET_GFX94X_DBSYNC",
```

Comment:
I highly recommend enabling also `'Fp32 Hip Debug gfx94X'` stage even is there is a lack of HW.

Reply:
btw this justifies renaming requested at https://github.com/ROCm/MIOpen/pull/2518#discussion_r1567442010